adoc/MAIN.release-notes.adoc

```diff
@@ -171,7 +173,60 @@ The `caaspctl` tool has been renamed from its working title to
+
+== Support and Life Cycle
+
+{productname} is backed by award-winning support from SUSE, an established technology leader with a proven history of delivering enterprise-quality support services.
+
+{productname} {productmajor} has a two-year life cycle. Each version will receive updates while it is current, and will be subject to critical updates for the remainder of its life cycle.
+
+For more information, check our Support Policy page https://www.suse.com/support/policy.html.
+
+== Support Statement for {productname} {LINK]REPORT BUG#
+To receive support, you need an appropriate subscription with SUSE. For more information, see https://www.suse.com/support/programs/subscriptions/?id=SUSE_CaaS_Platform.
+
+The following definitions apply:
+
+L1
+Problem determination, which means technical support designed to provide compatibility information, usage support, ongoing maintenance, information gathering and basic troubleshooting using available documentation.
+
+L2
+Problem isolation, which means technical support designed to analyze data, reproduce customer problems, isolate problem area and provide a resolution for problems not resolved by Level 1 or prepare for Level 3.
+
+L3
+Problem resolution, which means technical support designed to resolve problems by engaging engineering to resolve product defects which have been identified by Level 2 Support.
+
+For contracted customers and partners, {productname} {productmajor} is delivered with L3 support for all packages, except for the following:
+
+Technology Previews
+
+Packages that require an additional customer contract
+
+Packages with names ending in -devel (containing header files and similar developer resources) will only be supported together with their main packages.
+
+SUSE will only support the usage of original packages. That is, packages that are unchanged and not recompiled.
+
+== Documentation and Other Information
+
+=== Available on the Product Media 
+
+Read the READMEs on the media.
+
+Get the detailed change log information about a particular package from the RPM (where <FILENAME>.rpm is the name of the RPM):
+
+rpm --changelog -qp <FILENAME>.rpm
+Check the ChangeLog file in the top level of the media for a chronological log of all changes made to the updated packages.
+
+Find more information in the docu directory of the media of SUSE Linux Enterprise Server 15 SP1. This directory includes PDF versions of the SUSE Linux Enterprise Server 15 SP1 Installation Quick Start Guide.
+
+=== Externally Provided Documentation
+
+For the most up-to-date version of the documentation for {productname} {productmajor}, see https://www.suse.com/documentation/caas-platform-4.
+
+Find a collection of resources in the {productname} Resource Library: https://www.suse.com/products/caas-platform/#resources
 
+== Obtaining Source Code REPORT BUG#
```

Comment:
* ` REPORT BUG#` looks like leftover copypasta
* Feel free to copy this version of the section: https://gitlab.nue.suse.com/documentation/release-notes-sles/raw/12_SP5/adoc/source-code.adoc